### PR TITLE
PC-3178-Readme.md is updated

### PR DESCRIPTION
**Pull request by Yashu15380**

### CODE REVIEW

File: src/aks/README.md [hunk 1]
```diff
@@ -2,51 +2,60 @@
...
 
-Any images should be stored in the images directory, in a subdirectory named for the blueprint.
+### Environment : AKS Blueprint
 
 ![](images/aks.png)
 
 ## Prerequisites and Dependencies
 
-This section should list any prerequisites not covered by the blueprint.  For example, if the blueprint deploys into an existing VPC or requires a specific ami, outline that information in the prerequisites section.  Once a user has completed the tasks in the prerequisites section, they should be able to complete the blueprint deployment.
+- You will need azure_landing_zone_resource_group blueprint.
```
Comment (junaid18183) [inline src/aks/README.md:14]: need to run azure_landing_zone_resource_group or have resourcegroup pre-created

File: src/aks/README.md [hunk 2]
```diff
@@ -2,51 +2,60 @@
...
+- You will need azure_landing_zone_vnet blueprint.
```
Comment (junaid18183) [inline src/aks/README.md:15]: You will need to azure_landing_zone_vnet blueprint. or have vnet pre-created

File: src/aks/README.md [hunk 3]
```diff
@@ -2,51 +2,60 @@
...
+- You will need a Deploy provider of type Azure.
+- Also Dummy Deploy connection of type Winrm
```
Comment (junaid18183) [inline src/aks/README.md:17]: Its ssh not winRM

Comment (junaid18183) [reply src/aks/README.md:17]: If you are using private_cluster_enabled you need to connected to VDSS spoke or need to use VPN to access the k8s API

File: src/aks/README.md [hunk 4]
```diff
@@ -2,51 +2,60 @@
...
+| location            | aws          | Cloud Provider for CIDR                                      |
+| resource_group_name | null         | Specifies the Resource Group where the Managed Kubernetes Cluster should exist |
+| client_id           | null         | name or id of client|
+| client_secret       | null         | Client secrets|
+| ap_node_count       | 2            | node count for agent pool|
+| ap_vm_size          | Standard_A3  | volume size for agent pool|
+| ap_os_disk_size_gb  | 100          | os disk size in gb for agent pool| 
+| cluster_version     | 1.15.12      | kubernetes cluster version |
+| private_cluster_enabled| true      | If the cluster has the Kubernetes API only exposed on internal IP addresses|
+| vnet_subnet_id      | null         | The ID of the Subnet where the Agents in the Pool are provisioned |
```
Comment (junaid18183) [inline src/aks/README.md:47]: Make sure to verify all Inputs are added

Comment (Yashu15380) [reply src/aks/README.md:48]: all inputs added

**Pull request merged**